lib/hexpm_web/controllers/package_report_controller.ex

```diff
@@ -73,18 +90,31 @@ defmodule HexpmWeb.PackageReportController do
     report = PackageReports.get(params["id"])
     for_moderator = Users.has_role(conn.assigns.current_user, "moderator")
     user = conn.assigns.current_user
+    for_owner = Owners.get(report.package, user) != nil
 
     if report == nil do
       conn
       |> put_flash(:error, @report_not_accessible)
       |> put_status(400)
       |> redirect(to: Routes.package_path(HexpmWeb.Endpoint, :index))
     else
+      has_comments = Comments.count(report.id) != 0
+
+      comments =
+        if has_comments do
+          Comments.all_for_report(report.id)
+        else
+          []
+        end
```

Comment:
This is not an optimization. It's faster to just call `Comments.all_for_report(report.id)`.